DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -32,6 +32,21 @@ databricksClusterNodeType=Standard_DS3_v2
 #Databricks SKU
 databricksSku=premium
 
+# HDInsight cluster version.
```

Comment:
would it be a good idea to mention what DataX supports? 3.6 and 4.0.

Reply:
And also have we tested 4.0 on a non-whitelisted subscription?

Reply:
yes, i have tested the HDI4.0 with a non-whitelisted subscription (DataX Development Bugbash 2). No issue was found.